### PR TITLE
Check for ansible-lint version if not using an EE

### PR DESCRIPTION
**Pull request by shatakshiiii**

### CODE REVIEW

File: src/ansible_navigator/configuration_subsystem/navigator_post_processor.py [hunk 1]
```diff
@@ -655,6 +656,24 @@ def lintables(
...
+                except subprocess.CalledProcessError:
+                    exit_messages.append(
+                        ExitMessage(
+                            message=("ansible-lint does not seem to be installed correctly.")
+                        ),
+                    )
+                    exit_messages.append(
+                        ExitMessage(
+                            message=(
+                                "Ensure the command `ansible-lint --version` can be run prior to"
```
Comment (priyamsahoo) [inline src/ansible_navigator/configuration_subsystem/navigator_post_processor.py:671]: Can we make this message more generic? Something like, `Ensure that ansible-lint is installed in the environment`?

Comment (cidrblock) [reply src/ansible_navigator/configuration_subsystem/navigator_post_processor.py]: I prefer the original, this check is follow on to finding the executable, to ensure not only that is it installed, but it is a working installation.

Comment (priyamsahoo) [reply src/ansible_navigator/configuration_subsystem/navigator_post_processor.py]: Okay. @shatakshiii, I think you have to revert the changes :)

**Review by priyamsahoo — APPROVED** [review]

File: src/ansible_navigator/configuration_subsystem/navigator_post_processor.py (hunk 1 above)
Comment (shatakshiiii) [reply src/ansible_navigator/configuration_subsystem/navigator_post_processor.py:671]: reverted that change :)

**Pull request merged**